kivy/core/camera/camera_avfoundation_implem.m

```diff
@@ -158,6 +272,52 @@ - (CameraFrame*)getOutput;
     [localpool drain];
 }
 
+bool Camera::attemptFrameRateSelection(int desiredFrameRate){
+    bool isFPSSupported = NO;
+    AVCaptureDeviceFormat *currentFormat = [mCaptureDevice activeFormat];
+    for ( AVFrameRateRange *range in currentFormat.videoSupportedFrameRateRanges ) {
+        if ( range.maxFrameRate >= desiredFrameRate && range.minFrameRate <= desiredFrameRate )        {
+            isFPSSupported = YES;
+            break;
+        }
+    }
+
+    if( isFPSSupported ) {
+        if ( [mCaptureDevice lockForConfiguration:NULL] ) {
+            mCaptureDevice.activeVideoMaxFrameDuration = CMTimeMake( 1, desiredFrameRate );
+            mCaptureDevice.activeVideoMinFrameDuration = CMTimeMake( 1, desiredFrameRate );
+            [mCaptureDevice unlockForConfiguration];
+        }
+    } else {
+        NSLog(@"Selected FPS not available on this platform.");
```

Comment:
might want to add the value in the log, in case the dev didn't set a fixed value, but tried to have code find the proper value, might be useful to debug.

Reply:
Nice catch, added it.